Update translation_cn.csv

JesseZhou-1 opened the pull request:
I finished the Chinese translation, @WillemJiang could you help review it, thanks!

JesseZhou-1 committed:
Update translation_cn.csv

tapjdey commented:
Thanks @JesseZhou-1. Do you have any comments about rephrasing the questions as you had mentioned earlier?
If you have a proposal, could you add it to this PR?

JesseZhou-1 committed:
modified the expression of survey question according to the previous discuss

remove the CONCERNS, simply use "not at all fits" and "perfectly fits" to measure the the level of attitude

JesseZhou-1 commented:
hi, I just modified the question as we discussed earlier, I also revised the translation accordingly.

tapjdey commented:
Thanks!
